server/sonar-db-dao/src/main/java/org/sonar/db/ce/CeQueueDao.java

```diff
@@ -84,6 +85,10 @@ public int countByQuery(DbSession dbSession, CeTaskQuery query) {
     return mapper(dbSession).selectPendingByMinimumExecutionCount(minExecutionCount);
   }
 
+  public void resetTasksWithUnknownWorkerUUIDs(DbSession dbSession, Set<String> knownWorkerUUIDs) {
+    mapper(dbSession).resetTasksWithUnknownWorkerUUIDs(knownWorkerUUIDs, system2.now());
```

Comment:
unit tests for this new DAO method are missing
(yes, they are very similar to the one of `InternalCeQueueImpl#resetTasksWithUnknownWorkerUUIDs` but they must be written)